### DISCUSSION TITLE
2023/10/05/acm/codeforces/CodeforcesRound900/

### DISCUSSION THREAD

**Discussion by giscus** [Hukeqing/Hukeqing.github.io#9] [General(讨论贴)]
# 2023/10/05/acm/codeforces/CodeforcesRound900/

A. How Much Does Daytona Cost?大致题意给出一个数组和一个数字，问数组内是否存在某个子区间，使得给出的值出现次数最多 思路只有一个值也是子区间，只有它出现也是出现，所以只需要判断是否存在即可 AC code123456789101112131415void solve() &#123;    int _;    cin >> _;    for (int t

https://blog.mauve.icu/2023/10/05/acm/codeforces/CodeforcesRound900/

**Comment by kernelbin** [comment]
好厉害！

**Reply by Hukeqing** [reply]
没有你厉害哦～
